mac_build/buildMacBOINC-CI.sh

```diff
@@ -257,3 +257,30 @@ fi
 echo "Building ${target}...done"
 
 cd ..
+
+set -e
```

Comment:
Will this affect build script in any negative way if put this at the very beginning of the file?